src/classes/PanelData.js

```diff
@@ -71,7 +76,7 @@ class PanelData {
 		this._panelPort = port;
 		this._mountedComponents.panel = true;
 
-		getActiveTab((tab) => {
+		function tabCallback(tab) {
```

Comment:
@wlycdgr Double check this is ok

Reply:
Looks good/same when `paramTabId` is undefined and we end up calling `getActiveTab` as before. How come we need to check for the presence of that param - is it not the same as the active tab on Android?

Reply:
this was to allow loading of the panel in a browser tab for automation testing